api/api.go

```diff
@@ -35,17 +37,22 @@ type JobService interface {
 func CreateImportAPI(router *mux.Router, dataStore datastore.DataStorer, secretKey string, jobService JobService) *ImportAPI {
 
 	api := ImportAPI{dataStore: dataStore, router: router, jobService: jobService}
-	auth := NewAuthenticator(secretKey, "internal-token")
+	auth := NewAuthenticator(secretKey, "Internal-Token")
 
 	// External API for florence
-	api.router.Path("/jobs").Methods("POST").HandlerFunc(api.addJob)
-	api.router.Path("/jobs").Methods("GET").HandlerFunc(api.getJobs).Queries()
-	api.router.Path("/jobs/{id}").Methods("GET").HandlerFunc(api.getJob)
-	api.router.Path("/jobs/{id}").Methods("PUT").HandlerFunc(auth.ManualCheck(api.updateJob))
-	api.router.Path("/jobs/{id}/files").Methods("PUT").HandlerFunc(api.addUploadedFile)
+	api.router.Path("/jobs").Methods("POST").HandlerFunc(auth.Check(api.addJob))
```

Comment:
consider adding tests to check that each endpoint actually has authentication applied

Reply:
I added a set of tests to check the authentication is applied.